src/hearings/components/linked-hearings-case-title/linked-hearings-case-title.component.html

```diff
@@ -0,0 +1 @@
+<span class="govuk-caption-l">Link hearings to {{caseName}} {{caseId}} (full hearing)</span>
```

Comment:
caseId probably needs to transform to format with '1111-2222-3333-4444' as prototype shows.